frontend/app/components/landing-page-link-generator.tsx

```diff
@@ -0,0 +1,20 @@
+
+
+
+export function GenerateLandingPageLink({ title, description, buttonUrl, buttonText }) {
+
+    return (
+        <div class="row panel panel-default">
+            <div class="mrgn-tp-md header-bg">
+                <h2 id="a1-4" class="mrgn-tp-0">{title}</h2>
+                <div class="brdr-bttm-fade"></div>
+            </div>
+            <p class="mrgn-tp-md">{description}</p>
+            <li class="mrgn-bttm-sm xs-block">
+                <a class="btn btn-default btn-block" href={buttonUrl} >
```

Comment:
FYI use `className` instead of `class`, the latter of which is a reserved JS keyword.

Reply:
I've applied required changes, please review it, thanks